### PR TITLE
Fix: update application.properties.example

### PR DESCRIPTION
**Pull request by tobehardest**
Fix: application.properties.example is different from application.properties
ref: #10438 

## What is the purpose of the change

XXXXX

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [ ] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [ ] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

**Review by KomachiSion — APPROVED** [review]

File: distribution/conf/application.properties.example [hunk 1]
```diff
@@ -150,11 +152,12 @@ nacos.core.auth.enabled=false
...
+### worked when nacos.core.auth.system.type=nacos
 ### The token expiration in seconds:
-nacos.core.auth.default.token.expire.seconds=18000
-
-### The default token:
-#nacos.core.auth.default.token.secret.key=SecretKey012345678901234567890123456789012345678901234567890123456789
+nacos.core.auth.plugin.nacos.token.cache.enable=false
+nacos.core.auth.plugin.nacos.token.expire.seconds=18000
+### The default token (Base64 String):
+nacos.core.auth.plugin.nacos.token.secret.key=SecretKey012345678901234567890123456789012345678901234567890123456789
```
Comment (KomachiSion) [inline distribution/conf/application.properties.example:160]: Remove the default value.

**Comment by tobehardest** [comment]
Fixed.

**Comment by KomachiSion** [comment]
I see the file change is much different with last commit. Only change one line. Is there something wrong?

**Comment by tobehardest** [comment]
Sorry, I changed my computer and made a mistake, is it ok now?

**Review by KomachiSion — APPROVED** [review]

**Comment by KomachiSion** [comment]
@tobehardest Thanks, PR merged.

**Pull request merged**